src/android/unregisterNativeModule.js

```diff
@@ -1,75 +1,52 @@
-const readFile = require('./fs').readFile;
-const writeFile = require('./fs').writeFile;
-const path = require('path');
-const compose = require('lodash').flowRight;
+const fs = require('fs');
 const getReactVersion = require('../getReactNativeVersion');
 const getPrefix = require('./getPrefix');
 const isInstalled = require('./isInstalled');
 
-const cut = (scope, pattern) =>
-  scope.replace(pattern, '');
+const revokePatch = require('./patches/revokePatch');
+const makeSettingsPatch = require('./patches/makeSettingsPatch');
+const makeBuildPatch = require('./patches/makeBuildPatch');
+const makeStringsPatch = require('./patches/makeStringsPatch');
 
-module.exports = function unregisterNativeAndroidModule(name, dependencyConfig, projectConfig) {
-  const prefix = getPrefix(getReactVersion(projectConfig.folder));
-
-  /**
-   * @param  {String} content Content of the Settings.gradle file
-   * @return {String}         Patched content of Settings.gradle
-   */
-  const cutModuleFromSettings = (name) => (content) =>
-    cut(content, `include ':${name}'\n` +
-      `project(':${name}').projectDir = ` +
-      `new File(rootProject.projectDir, '../node_modules/${name}/android')\n`
-    );
-
-  /**
-   * Cut module compilation from the project build
-   * @param  {String} content Content of the Build.gradle file
-   * @return {String}         Patched content of Build.gradle
-   */
-  const cutModuleFromBuild = (name) => (content) =>
-    cut(content, `    compile project(':${name}')\n`);
-
-  const getAddPackagePatch = require(`./${prefix}/addPackagePatch`);
+module.exports = function unregisterNativeAndroidModule(
+  name,
+  androidConfig,
+  projectConfig
+) {
```

Comment:
I changed it because it was > 80 chars, so I to fit our styleguide, I moved some parts to the next strings.